src/commands/setheaderrowcommand.js

```diff
@@ -60,8 +60,10 @@ export default class SetHeaderRowCommand extends Command {
 	 * When the selection is already in a header row, it will set `headingRows` so the heading section will end before that row.
 	 *
 	 * @fires execute
+	 * @params {Boolean} [forceValue] If set, the command will set (`true`) or unset (`false`) header rows according to `forceValue`
+	 * parameter instead of the current model state.
 	 */
-	execute() {
+	execute( forceValue = null ) {
```

Comment:
It could be `execute( value )` or `execute( { value } )`. Note that `value` is essentially the same as `forceValue`. For me, `forceValue` makes more sense if usually the `value` is taken basing on the editor state. Also, as I wrote above, we already used that name.